rcl/src/rcl/arguments.c

```diff
@@ -0,0 +1,377 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rcl/arguments.h"
+
+#include <string.h>
+
+#include "rcl/error_handling.h"
+#include "rcl/validate_topic_name.h"
+#include "rcutils/logging_macros.h"
+#include "rcutils/strdup.h"
+#include "./arguments_impl.h"
+#include "./remap_impl.h"
```

Comment:
Sorted in d08034b87d737c0d0f7c2a95645cf5c55748e415. I did the `./` includes above instead of separating them with a blank line because it's consistent with what was done in `rcl.c`